app/controllers/v0/triage_teams_controller.rb

```diff
@@ -3,10 +3,11 @@ module V0
   class TriageTeamsController < SMController
     def index
       resource = client.get_triage_teams
-      resource = resource.sort(params[:sort])
-
       raise Common::Exceptions::InternalServerError unless resource.present?
 
+      resource = resource.sort(params[:sort])
+      resource = resource.paginate(pagination_params)
```

Comment:
this just to warn / inform, resource might be a method or scope in the controller, so this local variable might in affect override that method within the scope of this index action. But I don't think it will be a big deal (even if that is a thing).